calico_node/startup/autodetection/reachaddr.go

```diff
@@ -0,0 +1,50 @@
+package autodetection
+
+import (
+	"fmt"
+	gonet "net"
```

Comment:
nit: not a fan of renaming Go's native libraries. We implemented our "net" differently so our `/lib/net` should be imported as `cnet` or something. Most people reading the code will think of Go's native `net` when they see `net`